libs/angular-integration-interface/mocks/portal-message-service-mock.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core'
+import { FakeTopic } from './fake-topic'
+import { Message, PortalMessageService } from '../src/lib/services/portal-message.service'
+
+export function providePortalMessageServiceMock() {
+  return [PortalMessageServiceMock, { provide: PortalMessageService, useExisting: PortalMessageServiceMock }]
+}
+@Injectable({ providedIn: 'any' })
+export class PortalMessageServiceMock {
+  message$ = new FakeTopic<Message>()
```

Comment:
Wrong message. Should be from integration-interface -> both message classes must be imported